osu.Game/Input/IdleTracker.cs

```diff
@@ -40,7 +45,7 @@ public IdleTracker(double timeToIdle)
         protected override void Update()
         {
             base.Update();
-            isIdle.Value = TimeSpentIdle > timeToIdle;
+            isIdle.Value = TimeSpentIdle > timeToIdle && AllowIdle;
```

Comment:
I really don't think we want to be querying the containing input manager every frame.